### ISSUE TITLE
DJL:  Fast Food Store Distribution in Korea

### ISSUE THREAD

**Issue by djlee0202** [jsoma/playfair-projects#107]
I am changing the previous subject due lack of data.

There are 4 major fast food company in Korea. I would like to find the distribution of these fast food companies. The following is the chart of their market share.

![c29960e1d22d8fb8e980d052a2008f10](https://cloud.githubusercontent.com/assets/19519829/17483315/82346050-5d53-11e6-955b-444edf7f856d.jpg)

These are the location of the website which I need to parse the location of the stores.

Burgerking : http://www.burgerking.co.kr/
KFC: http://www.kfckorea.com/store/store_location.asp
Loterria : http://www.lotteria.com/Shop/Shop_Ajax.asp
McDonalds: http://www.mcdonalds.co.kr/www/kor/findus/district.do

# Pitch issue checklist

Paste this into the body of your pitch issue, NOT as the first comment! You can delete this line after you paste it.

- [x] My pitch issue has `[Pitch]` in the title
- [x] My pitch issue has `[Data request]` in the title if I need additional data
- [x] My pitch issue links to my data set
- [x] My pitch issue explains what I'd like to explore in the data set
- [x] My pitch issue includes multiple possible headlines
- [x] My pitch issue includes any links/images I'm using as inspiration
- [x] My pitch issue includes appropriate labels in the title such as `[Pitch]` and maybe `[Data Request]`
- [x] I have received two comments of peer feedback on my pitch issue
- [x] I have received editorial feedback on my pitch issue
- [ ] My pitch has been approved

**Comment by radhikapc** [comment]
Another story for foodies :-) In addition to replicating the finding, I would suggest you add your own angle, such as a different graph (sliding graph). Maybe try finding a new twist in the course of replicating the story. Check the link for a demo on sliding graph.

http://matplotlib.org/examples/widgets/slider_demo.html
http://stackoverflow.com/questions/6697259/interactive-matplotlib-plot-with-two-sliders

**Comment by honjy** [comment]
I don't think I want to read your story because it might put me off Chipotle forever, haha. I think it'd be cool if it could be an interactive bar chart though. In my mind there'd be a button you can slide along a calorie chart, showing you what you can order for 500 calories and what you can order for 600,000 calories haha.

**Comment by jsoma** [comment]
I feel like we need more specifics - "how it relates to consumer's health" is a liiiit bit broad. This is why we look for headlines on the checklist - it's easier to work your way through a dataset to prove/disprove/etc a headline than a general idea.

**Comment by playfairbot** [comment]
Closing pitch since story has been opened at #182
